src/pages/signin/TermsAndLicenses.js

```diff
@@ -31,7 +32,7 @@ const TermsAndLicenses = props => (
             .
         </Text>
         <View style={[styles.mt4, styles.alignItemsCenter, styles.mb2, styles.flexRow, styles.justifyContentBetween]}>
-            <LogoWordmark height={30} width={80} />
+            <LogoWordmark height={30} width={80} fill={colors.white} />
```

Comment:
Filling the wordmark is a clever idea but I wonder how exact the fill is or if we'll end up with a weird outline at times. @shawnborton what do you think of this implementation?

Reply:
@shawnborton friendly bump here

Reply:
Oh that's clever... I can't imagine it would have any weird outlines given that it's just an SVG shape, so I think this should be fine!